rules/style.js

```diff
@@ -0,0 +1,670 @@
+
+    // require multiline ternary
+    // https://eslint.org/docs/rules/multiline-ternary
+    // TODO: enable?
+    'multiline-ternary': ['off', 'never'],
+
+    // require a capital letter for constructors
+    'new-cap': [
+      'error',
+      {
+        newIsCap: true,
+        newIsCapExceptions: [],
+        capIsNew: false,
+        capIsNewExceptions: [
+          'Immutable.Map',
+          'Immutable.Set',
+          'Immutable.List',
+        ],
+      },
+    ],
+
+    // disallow the omission of parentheses when invoking a constructor with no arguments
+    // https://eslint.org/docs/rules/new-parens
+    'new-parens': ['error'],
+
+    // allow/disallow an empty newline after var statement
+    'newline-after-var': ['off'],
+
+    // https://eslint.org/docs/rules/newline-before-return
+    'newline-before-return': ['off'],
+
+    // enforces new line after each method call in the chain to make it
+    // more readable and easy to maintain
+    // https://eslint.org/docs/rules/newline-per-chained-call
+    'newline-per-chained-call': ['error', { ignoreChainWithDepth: 4 }],
+
+    // disallow use of the Array constructor
+    'no-array-constructor': ['error'],
+
+    // disallow use of bitwise operators
+    // https://eslint.org/docs/rules/no-bitwise
+    'no-bitwise': ['error'],
+
+    // disallow use of the continue statement
+    // https://eslint.org/docs/rules/no-continue
+    'no-continue': ['error'],
+
+    // disallow comments inline after code
+    'no-inline-comments': ['off'],
+
+    // disallow if as the only statement in an else block
+    // https://eslint.org/docs/rules/no-lonely-if
+    'no-lonely-if': ['error'],
+
+    // disallow un-paren'd mixes of different operators
+    // https://eslint.org/docs/rules/no-mixed-operators
+    'no-mixed-operators': [
+      'error',
+      {
+        // the list of arithmetic groups disallows mixing `%` and `**`
+        // with other arithmetic operators.
+        groups: [
+          ['%', '**'],
+          ['%', '+'],
+          ['%', '-'],
+          ['%', '*'],
+          ['%', '/'],
+          ['/', '*'],
+          ['&', '|', '<<', '>>', '>>>'],
+          ['==', '!=', '===', '!=='],
+          ['&&', '||'],
+        ],
+        allowSamePrecedence: false,
+      },
+    ],
+
+    // disallow mixed spaces and tabs for indentation
+    'no-mixed-spaces-and-tabs': ['error'],
+
+    // disallow use of chained assignment expressions
+    // https://eslint.org/docs/rules/no-multi-assign
+    'no-multi-assign': ['error'],
+
+    // disallow multiple empty lines, only one newline at the end, and no new lines at the beginning
+    // https://eslint.org/docs/rules/no-multiple-empty-lines
+    'no-multiple-empty-lines': ['error', { max: 1, maxBOF: 0, maxEOF: 0 }],
+
+    // disallow negated conditions
+    // https://eslint.org/docs/rules/no-negated-condition
+    'no-negated-condition': ['off'],
+
+    // disallow nested ternary expressions
```

Comment:
Add the link to the document of this rule.

```suggestion
    // disallow nested ternary expressions
    // https://eslint.org/docs/latest/rules/no-nested-ternary
```